chart/templates/xnat-web-statefulset.yaml

```diff
@@ -53,11 +59,33 @@ spec:
             - /bin/sh
             - -c
             - |
-              /usr/local/bin/wait-for-postgres.sh
               /usr/local/tomcat/bin/catalina.sh run
       initContainers:
+        - name: wait-for-postgres
+          image: {{ include "xnat.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+            - name: PGPASSWORD
+              {{- if .Values.postgresql.auth.existingSecret }}
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.postgresql.auth.existingSecret }}
+                  key: userPasswordKey
```

Comment:
based on the `secretKeys` [defined in the chart values](https://github.com/bitnami/charts/blob/6fef27c5c4c0da7ec01455bb332ecfc92535b5fc/bitnami/postgresql/values.yaml#L165-L172)

Reply:
I think the key should be `password` unless we are overriding the value of `userPasswordKey`. For example, the output of the secret created when setting `postgresql.auth.password` is (release name here is `xnat-core`):

```
Name:         xnat-core-postgresql
Namespace:    xnat-core
Labels:       app.kubernetes.io/instance=xnat-core
              app.kubernetes.io/managed-by=Helm
              app.kubernetes.io/name=postgresql
              app.kubernetes.io/version=16.4.0
              helm.sh/chart=postgresql-15.5.28
Annotations:  meta.helm.sh/release-name: xnat-core
              meta.helm.sh/release-namespace: xnat-core

Type:  Opaque

Data
====
postgres-password:  8 bytes
password:           4 bytes
```

Reply:
ah thanks for spotting that! I've set it to `key: {{ .Values.postgresql.auth.secretKeys.userPasswordKey }}` but could hardcode it as `password` if we don't plant to change the key?